pack/ur.json

```diff
@@ -5,6 +5,17 @@
         "deck_limit": 1,
         "faction_code": "anarch",
         "flavor": "Please, let me have this dream.\nFlip side:\nI'm going to be my own kind of hero.",
```

Comment:
Probably the flavor should be edited to remove the "flip side" notation?

Reply:
it should, but I am going to hold off on that until NRDB can handle the flip side data.

If i make this change right now, this text will simply disappear from NRDB, sadly.